lib/charms/magma_orchestrator_interface/v0/magma_orchestrator_interface.py

```diff
@@ -238,3 +238,54 @@ def _on_relation_changed(self, event: RelationChangedEvent) -> None:
             fluentd_address=remote_app_relation_data["fluentd_address"],
             fluentd_port=int(remote_app_relation_data["fluentd_port"]),
         )
+
+
+class OrchestratorProvides(Object):
```

Comment:
Isn't an example for the `provider` needed in the dosctring above?

Reply:
Good catch, done